### PR TITLE
Turned off payments validation

### PR DESCRIPTION
**Pull request by esuwu**

### CODE REVIEW

File: pkg/ride/functions_proto.go [hunk 1]
```diff
@@ -139,12 +132,12 @@ func reentrantInvoke(env Environment, args ...rideType) (rideType, error) {
...
 	address, err := env.state().NewestRecipientToAddress(recipient)
 	if err != nil {
-		return nil, errors.Errorf("failed to get address from dApp, invokeFunctionFromDApp")
+		return nil, errors.Errorf("reentrantInvoke: failed to get address from dApp, invokeFunctionFromDApp")
 	}
 	env.setNewDAppAddress(*address)
 	err = ws.smartAppendActions(paymentActions, env)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to apply attachedPayments")
+		return nil, errors.Wrapf(err, "reentrantInvoke")
```
Comment (nickeskov) [inline pkg/ride/functions_proto.go:140]: On my mind this message should be more informative

File: pkg/ride/functions_proto.go [hunk 2]
```diff
@@ -154,9 +147,8 @@ func reentrantInvoke(env Environment, args ...rideType) (rideType, error) {
 	}
 
 	res, err := invokeFunctionFromDApp(env, recipient, fnName, listArg)
-
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to get Result from invokeFunctionFromDApp")
+		return nil, errors.Wrapf(err, "reentrantInvoke")
```
Comment (nickeskov) [inline pkg/ride/functions_proto.go:151]: The same as above

File: pkg/ride/functions_proto.go [hunk 3]
```diff
@@ -203,15 +195,9 @@ func invoke(env Environment, args ...rideType) (rideType, error) {
...
-			return nil, errors.New("invoke: address and alias are nil")
-		}
-		addressFromAlias, err := env.state().NewestAddrByAlias(*recipient.Alias)
-		if err != nil {
-			return nil, errors.Errorf("invoke: failed to get address by alias, %v", err)
-		}
-		recipient = proto.NewRecipientFromAddress(addressFromAlias)
+	recipient, err = ensureRecipientAddress(env, recipient)
+	if err != nil {
+		return nil, errors.Wrap(err, "invoke")
```
Comment (nickeskov) [inline pkg/ride/functions_proto.go:200]: The same again

File: pkg/ride/functions_proto.go [hunk 4]
```diff
@@ -298,16 +284,19 @@ func invoke(env Environment, args ...rideType) (rideType, error) {
...
 	address, err := env.state().NewestRecipientToAddress(recipient)
 	if err != nil {
-		return nil, errors.Errorf("failed get address from dApp, invokeFunctionFromDApp")
+		return nil, errors.Errorf("invoke: failed get address from dApp, invokeFunctionFromDApp")
 	}
 	env.setNewDAppAddress(*address)
 	err = ws.smartAppendActions(paymentActions, env)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to apply attachedPayments")
+		return nil, errors.Wrapf(err, "invoke")
```
Comment (nickeskov) [inline pkg/ride/functions_proto.go:292]: Same

File: pkg/ride/functions_proto.go [hunk 5]
```diff
@@ -316,11 +305,8 @@ func invoke(env Environment, args ...rideType) (rideType, error) {
 	}
 
 	res, err := invokeFunctionFromDApp(env, recipient, fnName, listArg)
-
-	ws.blackList = ws.blackList[:len(ws.blackList)-1] // pop
-
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to get Result from invokeFunctionFromDApp")
+		return nil, errors.Wrapf(err, "invoke")
```
Comment (nickeskov) [inline pkg/ride/functions_proto.go:309]: Same

File: pkg/ride/functions_proto.go (hunk 2 above)
Comment (alexeykiselev) [reply pkg/ride/functions_proto.go:151]: Not in this case, the errors of `invokeFunctionFromDApp` is quite descriptive, so we have only to add where it happens. Previous error message was not correct.

**Review by nickeskov — APPROVED** [review]

**Pull request merged**